C-Plus-Plus/dp/SlidingWindowTechnique.cpp

```diff
@@ -0,0 +1,44 @@
+#include <algorithm>
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+long long maxSum(const vector<int> &arr, size_t k)
+{
+  size_t n = arr.size();
```

Comment:
I guess a lot of beginner programmers don't know about size_t type. It will be better if you can use any other data type or add a description of size_t in commented form.